libseccomp/src/lib.rs

```diff
@@ -250,6 +250,106 @@ impl From<&ScmpArgCompare> for scmp_arg_cmp {
     }
 }
 
+#[rustfmt::skip]
+#[doc(hidden)]
+#[macro_export]
+macro_rules! __private_scmp_cmp_arg {
+    (arg0) => { 0 };
+    (arg1) => { 1 };
+    (arg2) => { 2 };
+    (arg3) => { 3 };
+    (arg4) => { 4 };
+    (arg5) => { 5 };
+    (arg6) => { 6 };
```

Comment:
argument is up to 6, so `arg6` is not used.

Reply:
oops, counting starts at 0, right.